### PR TITLE
lint: Fix typos flagged by codespell

### PR DESCRIPTION
**Pull request by Munkybooty**

### CODE REVIEW

**Comment by Munkybooty** [comment]
All of the changes were found by running lint-spelling.py

File: contrib/macdeploy/gen-sdk [hunk 1]
```diff
@@ -57,8 +57,8 @@ def run():
         out_sdktgz_path = pathlib.Path("./{}.tar.gz".format(out_name))
 
     def tarfp_add_with_base_change(tarfp, dir_to_add, alt_base_dir):
-        """Add all files in dir_to_add to tarfp, but prepent MEMBERPREFIX to the files'
-        names
+        """Add all files in dir_to_add to tarfp, but prepent MEMBERPREFIX to the file's
+        name
```
Comment (UdjinM6) [inline contrib/macdeploy/gen-sdk:61]: I'm not sure this needs any fixing

File: src/llmq/instantsend.h [hunk 2]
```diff
@@ -160,7 +160,7 @@ class CInstantSendDb
      * Called when a ChainLock invalidated a IS Lock, removes any chained/children IS Locks and the invalidated IS Lock
      * @param islockHash IS Lock hash which has been invalidated
      * @param txid Transaction id associated with the islockHash
-     * @param nHeight height of the block which recieved a chainlock and invalidated the IS Lock
+     * @param nHeight height of the block which received a chainlock and invalidated the IS Lock
```
Comment (UdjinM6) [inline src/llmq/instantsend.h:163]: might also want to fix `src/llmq/instantsend.h:199: veryfy ==> verify`

File: src/crc32c/src/crc32c_sse42.cc [hunk 3]
```diff
@@ -176,7 +176,7 @@ uint32_t ExtendSse42(uint32_t crc, const uint8_t* data, size_t size) {
     }
   }
 
-  // Proccess the data in predetermined block sizes with tables for quickly
+  // Process the data in predetermined block sizes with tables for quickly
```
Comment (UdjinM6) [inline src/crc32c/src/crc32c_sse42.cc:179]: should not touch 3rd-party modules

File: src/ctpl_stl.h [hunk 4]
```diff
@@ -184,7 +184,7 @@ namespace ctpl {
         }
 
         // run the user's function that excepts argument int - id of the running thread. returned value is templatized
-        // operator returns std::future, where the user can get the result and rethrow the catched exceptins
+        // operator returns std::future, where the user can get the result and rethrow the caught exceptions
```
Comment (UdjinM6) [inline src/ctpl_stl.h:187]: same

File: src/cxxtimer.hpp [hunk 5]
```diff
@@ -60,7 +60,7 @@ class Timer {
      * Transfer constructor.
      *
      * @param   other
-     *          The object to be transfered.
+     *          The object to be transferred.
```
Comment (UdjinM6) [inline src/cxxtimer.hpp:63]: same

**Review by UdjinM6 — CHANGES_REQUESTED** [review]

File: contrib/macdeploy/gen-sdk (hunk 1 above)
Comment (Munkybooty) [reply contrib/macdeploy/gen-sdk:61]: the original just read a little strange and just adding the apostrophe on its own as codespell suggested wouldn't remedy the issue. So this is more for easier readability

**Review by PastaPastaPasta — APPROVED** [review]
utACK for squash merge

**Review by UdjinM6 — APPROVED** [review]
utACK

**Pull request merged**